### DISCUSSION TITLE
Updated to V4.1 and now the CC model doesn't solve

### DISCUSSION THREAD

**Discussion by nrz22** [NatLabRockies/floris#1003] [General]
Hello, I recently updated my Floris using a pip update and then tried to run a model that was working before, and now the CC model seems to run forever. I have been letting in run now for a few hours just to see, but both the GCH and Turbopark models ran in about 20 mins. Please let me know if you know of anything or have any suggestions. Thanks for your help. 

Cheers, 
Nicholas

**Comment by misi9170** [comment]
Hi @nrz22 ,

This is not an issue we've come across yet---I'll need a bit more information to help you out here.
- Can you supply a minimal example of what doesn't seem to be working from you?
- Which version did you update to v4.1 from (what was the last version you used that was working)?
- Could you try running the first example ([001_opening_floris_and_computing_power.py](https://github.com/NREL/floris/blob/v4.1.1/examples/001_opening_floris_computing_power.py)) but with line 22 changed to `fmodel = FlorisModel("inputs/cc.yaml")` and report on the results? I get the following output:
```
The turbine power matrix should be of dimensions 4 (n_findex) X 2 (n_turbines)
[[1753.95445918  282.99704736]
 [1753.95445918  282.99704736]
 [3417.79700509  600.24420452]
 [3417.79700509  600.24420452]]
Shape:  (4, 2)
The farm power should be a 1D array of length 4 (n_findex)
[2036.95150654 2036.95150654 4018.04120961 4018.04120961]
Shape:  (4,)
```
